crates/bevy_hierarchy/src/child_builder.rs

```diff
@@ -166,6 +168,36 @@ impl<'w, 's, 'a> ChildBuilder<'w, 's, 'a> {
 pub trait BuildChildren {
     /// Creates a [`ChildBuilder`] with the given children built in the given closure
     fn with_children(&mut self, f: impl FnOnce(&mut ChildBuilder)) -> &mut Self;
+    /// Creates a [`ChildBuilder`] with the given children built in the given closure
+    ///
+    /// Compared to [`with_children`][BuildChildren::with_children], this method returns the
```

Comment:
This is a very useful line. Can you add matching docs on with_children?

Reply:
Done!